tests/conftest.py

```diff
@@ -290,3 +291,9 @@ def pytest_runtest_setup(item):
                 pytest.skip(
                     f"\nTest {item.nodeid} only runs with {allowed_interfaces} interfaces(s) but {b} interface provided",
                 )
+
+
+# General DefaultQubit2 fixture, for any number of workers
+@pytest.fixture(scope="module", params=[None, 1, 2])
+def qubit_device_2(request):
+    return DefaultQubit2(max_workers=request.param)
```

Comment:
I know we have used fixtures for devices in the past, but I would prefer to just use `DefaultQubit2(max_workers=max_workers)` in individual tests.

It will make it substantially easier to just copy and paste the test into a notebook or standalone file to debug.

Reply:
Done.